addons/medical_treatment/ACE_Medical_Treatment.hpp

```diff
@@ -566,7 +566,10 @@ class ADDON {
         // How long until the maximum effect is reached
         timeTillMaxEffect = 30;
         // How many of this type of medication can be in the system before the patient overdoses?
-        maxDose = 4;
+        maxSafeDose = 4;
+        // The number of doses over maxDose that there is chance to not overdose. Chance decreases as dose gets higher.
+        // Dose 4: Safe | Dose 5 and 6: Possible Overdose | Dose 7: Overdose
+        chanceDoses = 2;
```

Comment:
`chanceDoses` means even less than `riskDoses`.

`maxDoseVariation` or something similar is what we need. Variation might be a bit misleading, as it might be understood in the statistical way, or in a +- way.

Reply:
`maxDoseDeviation` seems suitable